packages/features/bookings/lib/handleNewBooking/getEventType.ts

```diff
@@ -11,6 +12,10 @@ const _getEventType = async ({
   eventTypeId: number;
   eventTypeSlug?: string;
 }) => {
+  if (!eventTypeId && !eventTypeSlug) {
+    throw new HttpError({ statusCode: 400, message: "Either eventTypeId or eventTypeSlug must be provided" });
+  }
+
   // handle dynamic user
   const eventType =
     !eventTypeId && !!eventTypeSlug ? getDefaultEvent(eventTypeSlug) : await getEventTypesFromDB(eventTypeId);
```

Comment:
Since we have a guard preceding this, I'd say we can simplify it further and make it cleaner
```suggestion
  const eventType = eventTypeId
  ? await getEventTypesFromDB(eventTypeId)
  : getDefaultEvent(eventTypeSlug);
```

Reply:
Oh Typescript!
<img src="https://media2.giphy.com/media/v1.Y2lkPWJkM2VhNTdlY2FxejFhYjNkMG93OW1wM2E4aXZ2d3N1OGZucmlnYWptdTczdHdwNCZlcD12MV9naWZzX3NlYXJjaCZjdD1n/snEeOh54kCFxe/giphy.gif"/>

Reply:
Maybe we could do 
```
const eventType = eventTypeId 
  ? await getEventTypesFromDB(eventTypeId) 
  : getDefaultEvent(eventTypeSlug!);
```

The non-null assertion should be similar to what we had earlier (!!eventTypeSlug when !eventTypeId). But we can revisit, non-blocking.